apps/bfd-model/bfd-model-rif/src/main/resources/db/migration/V96__recreate_pre_adj_child_tables.sql

```diff
@@ -0,0 +1,198 @@
+-- Rebuild the schema to match new RDA specifications and truncate tables to end soft launch
+
+drop table rda.fiss_audit_trails;
+drop table rda.fiss_diagnosis_codes;
+drop table rda.fiss_payers;
+drop table rda.fiss_proc_codes;
+drop table rda.mcs_adjustments;
+drop table rda.mcs_audits;
+drop table rda.mcs_details;
+drop table rda.mcs_diagnosis_codes;
+drop table rda.mcs_locations;
+truncate table rda.fiss_claims;
```

Comment:
don't we need to rebuild these two, I saw that you changed some varchar(1)/char(1) logic in the FissClaim.yaml and McsClaim.yaml